### PR TITLE
Sle 12 sp4

### PR DESCRIPTION
**Pull request by varkoly**
fixing bug#1099185 - yast2-auth-client incorrect version revert

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,3 +1,4 @@
-FROM yastdevel/ruby
+FROM yastdevel/ruby:sle12-sp4
```
Comment (jreidinger) [inline Dockerfile]: this is wrong, master should not be build for SP4

File: Rakefile [hunk 2]
```diff
@@ -1,5 +1,7 @@
 require "yast/rake"
 
+Yast::Tasks.submit_to :sle12sp4
```
Comment (jreidinger) [inline Rakefile]: and same here, master is for SP1 and not SP4, so please revert all these changes ( `git revert` is preffered way )

**Comment by jreidinger** [comment]
@varkoly if I get bug report, I think what you actually need is to cherry-pick some commit from `master` branch into `SLE12-SP4` branch. here you do opposite stuff, you try to merge SP4 into master which is TW/SLE15 SP1 targeted.

**Comment by varkoly** [comment]
Closed because it is a bad request.

**Pull request closed without merging**